### ISSUE TITLE
QA for Intent to File (Darby)

### ISSUE THREAD

**Issue by zslocum1** [department-of-veterans-affairs/va.gov-team#128647]
## Background

This ticket acts as a host for QA of the Intent to File (ITF) feature while also taking into account accessibility testing. Feel free to refer to this [Intent to File Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1766509810740/84d6573c9b85a6829a76917683e3e34b1c501d48) for assignments and guidance.

## Tasks
- Review the designated pathways (Both Veteran and Non-Veteran) via your specified devices
- Notate QA findings directly in the Mural — one card per finding
- Use the following OS/screen reader/browser combinations:

1. MacOS, Safari
2. Windows, Chrome
3. iOS, Safari

Note: The combination of **Android, Chrome** is currently out of scope as we do not have a device for testing that; we will pursue Android testing later when a device is obtained.

## Use of color and color contrast checks

- [ ] Test if text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [ ] Test if text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [ ] Test non-text elements for a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [ ] Confirm color is not the only way to distinguish links from other text (eg. links are underlined)
- [ ] Confirm any charts, maps, infographics, and tables convey all information without only relying on color
- [ ] Review content to ensure it does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Test color contrast ratios for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) Test all elements for retaining their meaning for all supported dark, contrast, and inverted color modes

## axe checks

- [ ] Each page or screen has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing
- [ ] axe DevTools scan results are documented

## Content zoom and reflow checks

- [ ] (Web only) Test page elements for readability and usability at 200% zoom
- [ ] (Web only) Test page elements for readability and usability at 300% zoom
- [ ] (Web only) Test page elements for readability and usability at 400% zoom
- [ ] (Mobile app only) Test page elements for readability and usability when device text is at the maximum possible size

## Keyboard navigation checks

- [ ] Test each link, button, form input, checkbox, radio button, select menu, and custom element for receiving keyboard focus
- [ ] Test each link, button, form input, checkbox, radio button, select menu, and custom element for responding to expected keys
- [ ] Test all elements under focus for a visible focus indicator
- [ ] Test the order of [Tab] stops to see if they make sense and are appropriate for completing tasks

## Mouse only/touchscreen checks

- [ ] Tasks do not depend on specific keyboard actions
- [ ] Actions do not depend on touch/click timings
- [ ] Using a mobile device, full functionality can be used with only one finger to gesture

## Acceptance Criteria

- [ ] You have reviewed the designated pathways and have reported any QA findings.

**Comment by darbyemadewell** [comment]
Testing completed using Mac/Google Chrome (a11y checks), Windows/Edge, and iOS/Google Chrome. Additional screen reader and voice control testing was completed for the failed submission state for the Veteran and non-Veteran pathways. Results have been added to the Mural for ITF QA.
